src/java/org/apache/cassandra/db/Directories.java

```diff
@@ -544,14 +544,21 @@ public static boolean hasDiskSpaceForCompactionsAndStreams(Map<File, Long> expec
      */
     public static boolean hasDiskSpaceForCompactionsAndStreams(Map<FileStore, Long> totalToWrite)
     {
+        boolean hasSpace = true;
         for (Map.Entry<FileStore, Long> toWrite : totalToWrite.entrySet())
         {
             long availableForCompaction = getAvailableSpaceForCompactions(toWrite.getKey());
             logger.debug("FileStore {} has {} bytes available, checking if we can write {} bytes", toWrite.getKey(), availableForCompaction, toWrite.getValue());
```

Comment:
as [the comment here](https://github.com/apache/cassandra/pull/2284#pullrequestreview-1393420403)
I think we can also remove this message .